db/migrate/20240503200211_add_request_units_to_item_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddRequestUnitsToItemRequests < ActiveRecord::Migration[7.0]
+  def change
+    add_column :item_requests, :reporting_unit, :string
+  end
```

Comment:
Change reporting_unit to request_unit here, please.      Reporting units would be what is on the annual reports, request units the unit that is requested,  yes?  (Take as read any follow-on changes from that *g*)